cheatsheets/PHP_Configuration_Cheat_Sheet.md

```diff
@@ -111,3 +107,13 @@ report_memleaks         = On
 track_errors            = Off
 html_errors             = Off
 ```
+
+## Suhosin (PHP 5.X)
+
+Consider using [Suhosin](https://suhosin.org/) if you want to patch many custom security flaws in various parts of PHP. 
+Note: only usable in 5.X branch and should not be used in prod now as 5.X is EOL.
+
+## Snuggleupagus (PHP 7.X)
+
+[Snuggleupagus](https://github.com/nbs-system/snuffleupagus) is continuing what [Suhosin started](https://github.com/sektioneins/suhosin-ng/wiki/News#2019-07-17-suhosin-is-back). It allows you to control PHP behaviour at a deeper level. [Detailed documentation is available here](https://snuffleupagus.readthedocs.io/)
```

Comment:
@rbsec is this project actually still being worked at? If it's in Beta, the development work is too slow. We don't set projects that are still in Beta in the CS as they are not good recommendations for people looking for robust recommendations that are quick. Can we look into something different, or this section will have to be removed.

Reply:
@ThunderSon there's been a reasonable number of commits in the GitHub repo this year, so it's still under development, but it's certainly in the pre-release stage. There was a Suhosin7 project, but that got abandoned a while ago. I don't think there's anything more mature than Snuffleupagus, but I'll have more of a look around and see if there's anything out there.

Reply:
Since there's still that much confusion on it and it's _that much early_, I'll refrain from putting this in the project.

Reply:
Ohk i think i get where the confusion is coming from.  Let me put things in a timeline.
PHP5 : suhosin
PHP 7: suhosin 7 starts but no work is done, Snuffleupagus comes in and does work. Snuffleupagus is usable and works.  suhousin-ng comes and suggests we will do our work but will push updates back to Snuffleupagus as they are doing good. 

The two links i gave were for this purpose to clearly suggest the old here suhosin is now supporting new kid Snuffleupagus

does that makes it clear. 

if you want i can remove all links and just keep Snuffleupagus in link.